flowzone.yml

```diff
@@ -3555,6 +3850,7 @@ jobs:
       - custom_publish
       - custom_always
       - cloudformation_finalize
+      - terraform_finalize
```

Comment:
These finalize jobs don't need to be here, since the requirement for All Jobs really only matters for PR branch rules.

I don't _think_ it will break, but it is certainly not required.

However `cloudformation_test` and `terraform_test` should be here. Maybe add a comment explaining the above so others don't make the same mistake.

Reply:
@klutchell before I added `cloudformation_finalize`, my PRs would merge prematurely while those jobs were still running. So at least one of these should be there for each project type, or I've missed something..

Reply:
the `cloudformation_test` job should be there, same for terraform, because finalize only happens after the PR is merged, why would we want to block the merge waiting for finalize jobs?

Reply:
All Jobs only exists so we can make it a required check for merges. It doesn't make any sense for post-merge jobs to be there, but all the test and deploy jobs should be there.